cad/scripts/diagnostics/diag_dump_part.py

```
@@ -248,7 +259,16 @@ def _feature_data(feat, type_name):
             if v is None:
                 continue
             out[prop] = _mm(v) if prop == "FromOffsetDistance" else v
-    elif type_name in ("RevolveBoss", "RevolveCut", "Revolve"):
+        for key, meth, args in (("wall_mm", "GetWallThickness", (True,)),
+                                ("cap_mm", "CapThickness", ())):
+            try:
+                v = getattr(d, meth)(*args) if args else _g(d, meth)
+            except Exception:  # noqa: BLE001
+                continue
```

Ruff (0.16.2)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)